yazilim-dilleri/javascript/index.js

```diff
@@ -0,0 +1,7 @@
+var chars = ["h","e","l","l","o"," ","w","o","r","l","d"];
```

Comment:
```suggestion
const chars = ['h','e','l','l','o',' ','w','o','r','l','d'];
```
Cunku mutate etmiyorsun.

Reply:
Single quote daha okunabilir hale getiriyor.